web_widget_text_count/readme/ROADMAP.rst

```diff
@@ -0,0 +1,4 @@
+Patch Text class on Python level to allow propagation of size same as in Char fields.
+After modify js to relies it on field attribute.
+
+Impossible to put counter field after Text field in one line in UI.
```

Comment:
~Impossible to put counter field after Text field in one line in UI.~
Support inline position of the counter next to the textarea.